eu_dgc_v1_schema_draft_07.json

```diff
@@ -88,20 +88,9 @@
                 },
                 "dob": {
                     "title": "Date of birth",
-                    "description": "Mandatory if no Person identifier is provided",
+                    "description": "The date of birth for of the person addressed in the certificate",
```

Comment:
minor: "The date of birth for of the person" -> "The date of birth of the person"